### PR TITLE
[AV2-75] Multiple lines on chat

### PR DESCRIPTION
**Pull request by sasangachathumal**
create a pipe to identify urls.
added text area to type message.
added some styles to show multi line messages.
created some new css variables.

### CODE REVIEW

**Comment by sasangachathumal** [comment]
This is have RUL identify pipe but it not use in this feature. because it have some issue with anchor tag color. it will be done in another story.

File: src/app/home/todo-card/todo-card.component.scss [hunk 1]
```diff
@@ -5,6 +5,19 @@
   background: #fff;
   min-height: 58px;
 }
+.todo-item {
```
Comment (shawnm0705) [inline src/app/home/todo-card/todo-card.component.scss]: This class has been removed already. Do we still need this CSS ?

File: src/theme/variables.scss [hunk 2]
```diff
@@ -5,6 +5,9 @@
 
 :root {
 
+  --practera-black: black;
```
Comment (shawnm0705) [inline src/theme/variables.scss]: We don't need to create a variable for this, just use `black` & `white` in CSS

File: src/app/home/todo-card/todo-card.component.scss (hunk 1 above)
Comment (sasangachathumal) [reply src/app/home/todo-card/todo-card.component.scss]: I use this to limit long description text in the todo items on dashboard.

Comment (sasangachathumal) [reply src/app/home/todo-card/todo-card.component.scss]: it was there i didn't add this. I have added this part.
`.icon-item {
    ion-text {
      p {
        max-height: 16px;
        -webkit-line-clamp: 1;
        line-height: 16px;
        overflow: hidden;
      }
    }
  }`

**Pull request merged**